scripts/generate-auth-params.js

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const crypto = require('crypto');
+
+const key =
+  process.env.APP_KEY ||
+  'd94c278085e49e807b67a58b8fcb92239b3fe881252f1938e3e0ca09611684b3790df3a22c08a32959309fdfcf074b4204c2f84a1ef03025621c4cf23a5521f24299ba857956a1fcf1b61715cb8154baeedba4e0409e636da9e91d7a618a4403e748c3a2';
+
+const username = process.env.APP_USER || 'admin@hel.fi';
+const resource = process.env.APP_RESOURCE;
```

Comment:
How are these env variables meant to be used? Should we add something about them in readme? Also should we default to some resource and organization? If these are meant to be user configured by his own, should we name these with hauki-prefix to separate them from other possible environment variables used by other apps?

Reply:
Hmm.., I do not think we will have other apps envs in our repo 🤔 
But I think we can use the HAUKI-prefix